components/Footer.tsx

```diff
@@ -26,7 +31,11 @@ const footerInfoArray = [
   },
 ]
 
-export default function Footer() {
+type Props = {
+  color: 'white' | 'black'
```

Comment:
I think this **props name** is not clear.
```suggestion
  theme: 'white' | 'black'
```
or
```suggestion
  fontColor: 'white' | 'black'
```
or
```suggestion
  variant: 'white' | 'black'
```

Reply:
changed to variant